src/VideoUtils.ts

```diff
@@ -24,14 +24,14 @@ function publishedTimeToString(date: string): string {
     const minutes: string = dateJs.getMinutes().toString();
     const seconds: string = dateJs.getSeconds().toString();
 
-    return `${hours}:${minutes}:${seconds}`;
+    return `${hours}.${minutes}.${seconds}`;
```

Comment:
For the future, let's never compute something in the return statement, always evaluate what you need to evaluate in a var before return, then return the var.

Reply:
Ok, will do in the future. I'm not sure about the reason for this, could you tell me?